.mocharc.js

```diff
@@ -21,5 +21,8 @@ module.exports = {
     '**/build/**',
     'docs/.next/**',
   ],
-  spec: ['packages/**/*.test.{js,ts,tsx}', 'docs/src/modules/**/*.test.{js,ts,tsx}'],
+  spec: [
+    'packages/{,!(x-charts)/**}/*.test.{js,ts,tsx}', // x-charts requires 'tsx/cjs' which conflict with the babel date-fns override for picker tests
+    'docs/src/modules/**/*.test.{js,ts,tsx}',
+  ],
```

Comment:
Have you considered adding `'packages/x-charts/**',` to the list of ignores and keeping the `spec` as is? 🤔